snmp/datadog_checks/snmp/data/conf.yaml.example

```diff
@@ -130,6 +136,16 @@ instances:
     #
     # retries: 5
 
+    ## @param discovery_interval - integer - optional - default: 3600
+    ## Interval between 2 discovery starts, in seconds.
+    #
+    # discovery_interval: 3600
+
+    ## @param discovery_pause - integer - optional - default: 1
+    ## Pause happening between each discovery steps, in seconds.
+    #
+    # discovery_pause: 1
```

Comment:
Why the pause is necessary and why 1 second (seems quite long) ?

Reply:
Answer above. The process doesn't need to be super fast at least for now.